### PR TITLE
AAP-15779 - Restructuring hub contents in preparation for docathon

### PR DESCRIPTION
**Pull request by dcdacosta**
This PR restructures the existing hub documentation into the 3 defined documents approved in the proposed outline and includes the following changes:

Restructure existing documents into 3 documents including:

- Getting started with automation hub
- Managing content in your private automation hub
- Ansible Automation Platform installation guide

Rename the 'modules/automation-hub' directory to 'modules/hub'

Update symlinks to automation-hub directory to hub

### CODE REVIEW

File: downstream/modules/hub/proc-basic-repo-sync.adoc [hunk 1]
```diff
@@ -13,7 +13,7 @@ All collections in the configured remote are downloaded to your custom repositor
 +
 [NOTE]
 ====
-To limit repository synchronization to specific collections within a remote, you can identify specific collections to be pulled using a requirements.yml file. See xref:proc-create-remote[Create a remote] for more information.
+To limit repository synchronization to specific collections within a remote, you can identify specific collections to be pulled using a requirements.yml file. See xref:proc-create-remote_remote-management[Create a remote] for more information.
```
Comment (ianf77) [inline downstream/modules/hub/proc-basic-repo-sync.adoc:16]: I'm assuming the additional _remote-management is the {Context}?

**Review by ianf77 — COMMENTED** [review]
One comment, don't know if it's relevant.

File: downstream/modules/hub/proc-basic-repo-sync.adoc (hunk 1 above)
Comment (dcdacosta) [reply downstream/modules/hub/proc-basic-repo-sync.adoc:16]: Hey @ianf77 , Correct! Those files appear in this document several times now and that xref was causing issues. I've noted it in the tracker, so we can go ahead and merge this.

**Review by ianf77 — APPROVED** [review]
LGTM

**Comment by ariordan-redhat** [comment]
Built `/titles/hub/getting-started` and `/titles/hub/managing-content`

**Pull request merged**